djangoapp/README.md

```diff
@@ -0,0 +1,12 @@
+# Reference Django app
+Functional skeleton for a Django app that employs secrets management and AWS resources
+Requirements: 
```

Comment:
I'd say that we should be recommending developers use docker-compose now, so we should provide a `docker-compose.yml` file rather than local machine install instructions.